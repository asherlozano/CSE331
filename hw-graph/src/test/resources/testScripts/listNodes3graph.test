# listNodes3graph test file
# tests if you the listNodes with 3 graphs
# listNodes lists all the nodes of 3 graph

# Create a graph
CreateGraph g1
CreateGraph g2
CreateGraph g3

# Create a node
AddNode g1 n1
AddNode g2 n1
AddNode g3 n1

# Print the graphs nodes
ListNodes g1
ListNodes g2
ListNodes g3