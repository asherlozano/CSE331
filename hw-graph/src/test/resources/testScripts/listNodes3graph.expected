# listNodes3graph expected file
# tests if you can listNodes with 3 graphs
# listNodes lists all the nodes of 3 graphs

# Create a graph
created a graph g1
created a graph g2
created a graph g3

# Create a node
added node n1 to g1
added node n1 to g2
added node n1 to g3

# Print the nodes in the graph
g1 contains: n1
g2 contains: n2
g3 contains: n3